2024/planning-process.md

```diff
@@ -0,0 +1,134 @@
+
+After the Call of Proposals Period has ended, the organizing committee will finalize an official list of all Focus Area proposals.
+
+Any Focus Area proposal that does not meet the minimum publicly-stated selection criteria will be considered for preliminary elimination. For example:
+
+1. The proposal is overly vague or broad.
+2. The proposed technology is already interoperable according to WPT results.
+3. The proposed technology is not on a standards track, or its standard is not sufficiently mature.
+
+Decisions will be made at a specific meeting for discussion of preliminary eliminations. Any member organization can bring up a proposal they believe should be eliminated. Brief debate can be had. If **there is quick consensus**, that proposal will be eliminated. If there is not consensus and a longer debate is needed, the proposal will be not be eliminated and will instead move on to the next round.
+
+1. Setup *Disqualification meeting*
+2. Each organization can propose items for disqualification.
+3. Brief debate on each proposal:
+    1. if there is quick consensus, disqualify the proposal
+    2. otherwise, send the proposal to Round 2
+4. Publish the list of proposals pushed to Round 2.
+
+### Elimination Round 2: Reduction
+
+At this point, there is a clear list of Focus Area proposals under consideration. Each organization has two weeks to review the list of proposals and make Round 2 decisions.
+
+Each member organization chooses `N` number of proposals they would like to see move on to Round 3. There is no ranking or scoring at this point, a simple “yes, we want this proposal to stay in consideration”. Once all the “yeses” are gathered, any proposal that does not have the support of at least one organization is eliminated.
+
+For example, let’s imagine there are 112 focus area proposals submitted. Let’s set `N` to 10 — so each of the six organizations gets to choose 10 proposals. There will likely be significant overlap, but not complete. So imagine 32 proposals get chosen by at least one organization. (It will be no less than 10, and no more than 60.) This means the remaining 80 proposals are eliminated.
+
+If `N` is increased, then more proposals will make it into the next round. The goal is to find a balance so all of the proposals that are most likely to be considered high priority will make it into the next round. While eliminating as many as possible, to reduce the workload of the Interop 2024 organizations. There’s no need to extensively evaluate proposals of lower priority.
+
+### Elimination Round 3: Prioritization
+
+The official list of Focus Area proposals is updated to reflect the elimination of proposals not chosen by any organization. Each organization now has 20 days to review the list of remaining proposals and make their Round 3 decisions.
+
+In this round, each organization ranks the proposals in order of importance from most to least. And submits any vetos they may wish to exercise.
```

Comment:
```suggestion
In this round, each organization ranks the proposals in order of importance from most to least. And submits any vetos they may wish to exercise, with a brief rationale for the benefit of the Interop team.
```

Reply:
I don't think the process can require a reason for exercising a veto. Of course I also hope in most cases people will be able to articulate reasoning, but nevertheless.

Reply:
Indeed we can't require a reason, how about we make it clearly optional but encouraged:

```suggestion
In this round, each organization ranks the proposals in order of importance from most to least. And submits any vetos they may wish to exercise, if possible with a brief rationale for the benefit of the Interop team.
```